File: 백준/Bronze/1145. 적어도 대부분의 배수/적어도 대부분의 배수.java
import java.util.Scanner;
public class Main {
    public static void func(int[] arr){
        int a = 1;
        while(true){
            int count = 0;
        for(int i = 0; i < 5; i++){
            if(a % arr[i] == 0) count++;
            if(count == 3){System.out.println(a); return;}
        }
        a++;
        }
    }
    public static void main(String[] args)
    {
        Scanner scanner = new Scanner(System.in);
        int[] numbers = new int[5];
        for(int i = 0; i < 5; i++)
            numbers[i] = scanner.nextInt();
        scanner.close();
        func(numbers);
    }
}

File: 백준/Bronze/1145. 적어도 대부분의 배수/적어도 대부분의 배수.cc
#include <bits/stdc++.h>
using namespace std;
int arr[5];
void func(int arr[]){
    int a = 1;
    while(1){
        int count = 0;
        for(int i = 0; i < 5; i++){
            if(a % arr[i] == 0) count++;
            if(count == 3){cout << a << '\n'; return;}
        }
        a++;
    }
}
int main(void){
    for(int i = 0; i < 5; i++)
        cin >> arr[i];
    func(arr);
    return 0;
}